_posts/2015-09-20-mikaila-waters-illustrator.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout:     post
+title:      "Illustrator for Developers and SVGs"
+subtitle:   "Illustrator basics and how to create an SVG icon"
+date:       2015-09-21 12:00:00
+author:     "Mikaila Waters"
+header-img: "img/illustrator.jpg"
+---
+
+## Background on Mikaila
+
+Mikaila is a current student in the Firehose Project curriculum and works at an edtech startup called Gradeable on all aspects of the product. She started at Gradeable as the designer, coming from a background in Architecture, and taught herself HTML, CSS, and Javascript.
+
+Mikaila walks you through the common tools she uses in Illustrator and how to create an SVG image for your front end. Get in touch with Mikaila [here](https://twitter.com/oh_my_kaila).
+
+{% include youtubePlayer.html id="8ZJZ0zjxGxk" %}
+
+
+Watch the full video [here](https://youtu.be/8ZJZ0zjxGxk).
```

Comment:
This should be the link to the full lightning talks video. Not a big issue at all everyone has done the exact same thing :smile: 

This is how it should look. 
`Watch the full video [here](https://youtu.be/rQ8iLTFZyz0)`